File: 0392-is-subsequence/0392-is-subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        int sp=0;
        int tp=0;
        int n=s.length();
        int m=t.length();
        while(tp<m && sp<n){
            if(s[sp]==t[tp]){
                sp++;
            }
            tp++;
        }
        if(sp==n){
            return true;
        }else{
            return false;
        }
    }
};

File: 0392-is-subsequence/0392-is-subsequence.py
class Solution(object):
    def isSubsequence(self, s, t):
        sp=0
        tp=0
        n=len(s)
        m=len(t)
        while(sp<n and tp<m):
            if(s[sp]==t[tp]):
                sp+=1
            tp+=1
        if(sp==n):
            return True
        else:
            return False